### PR TITLE
core/services/pg: simplify TxOptions

### PR DESCRIPTION
**Pull request by jmank88**
This PR removes some unused options from `package pg`. This will simplify the LOOPP migration.

### CODE REVIEW

**Review by patrickhuie19 — CHANGES_REQUESTED** [review]
LGTM, but on a minor note I think we'll have to remove 
* `defaultLockTimeout`
* `defaultIdleInTxSessionTimeout`

from `core/services/pg/utils.go`

**Comment by jmank88** [comment]
> LGTM, but on a minor note I think we'll have to remove
> 
> * `defaultLockTimeout`
> * `defaultIdleInTxSessionTimeout`
> 
> from `core/services/pg/utils.go`

Yay, the linter is finally working again.

**Review by patrickhuie19 — APPROVED** [review]

File: core/services/pg/transaction.go [hunk 1]
```diff
@@ -7,20 +7,16 @@ import (
...
-
 	"github.com/smartcontractkit/chainlink-relay/pkg/logger"
-
 	corelogger "github.com/smartcontractkit/chainlink/v2/core/logger"
 )
 
 type TxOptions struct {
 	sql.TxOptions
-	LockTimeout            time.Duration
-	IdleInTxSessionTimeout time.Duration
```
Comment (dimriou) [inline core/services/pg/transaction.go:23]: Curious as to why these were added if they are not utilized. Were they never needed or there was a time when we used to use them?

**Review by dimriou — APPROVED** [review]

File: core/services/pg/transaction.go (hunk 1 above)
Comment (jmank88) [reply core/services/pg/transaction.go:23]: IIRC they were used in the past :shrug:

**Pull request merged**